content/applications/general/auth/google.rst

```diff
@@ -2,38 +2,39 @@
 Google Sign-In Authentication
```

Comment:
For this page I'm not sure how helpful the numbered list instructions are vs. paragraph form.
Not a block for the review, however maybe we can entertain restructuring at some point.